app/assets/stylesheets/layout/page_specific/_category.scss

```diff
@@ -0,0 +1,53 @@
+// Category level pages
+
+.l-category__heading {
+  margin-top: 3.75rem;
+}
+
+.l-category__top {
+  border-bottom: 1px solid #edf0f0;
+  color: $color-true-black;
+  padding-bottom: 40px;
+  position: relative;
+}
+
+.l-category--image {
+  background-position: right 10px;
+  background-repeat: no-repeat;
+  background-size: 40% auto;
+  padding-bottom: 60px;
+
+  @include respond-to($mq-s) {
+    background-position: right 70px;
+  }  
+}
+
+p.l-category__top__intro {
```

Comment:
These seems a little odd. Could this be a child of a container called `.l-category__top__intro`?